src/common/processing_linux.c

```diff
@@ -126,10 +126,11 @@ void ffProcessGetInfoLinux(pid_t pid, FFstrbuf* processName, FFstrbuf* exe, cons
 
     #ifdef __linux__
 
-    char filePath[64];
-    snprintf(filePath, sizeof(filePath), "/proc/%d/cmdline", (int)pid);
+    char filePath[18];
+    snprintf(filePath, sizeof(filePath), "/proc/%d/", (int)pid);
+    FF_AUTO_CLOSE_FD int dfd = open(filePath, O_RDONLY | O_CLOEXEC);
```

Comment:
You saved only one cheap `snprintf` call but wasted two expensive syscalls (`open` and `close`). It's not worth the effort.

Reply:
Done.